packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -82,5 +89,6 @@ const findSelector = (component: (props: any) => JSX.Element | undefined): any =
     case EndCallButton:
       return emptySelector;
   }
-  throw 'Can\'t find corresponding selector for this component. Please check the supported components from Azure Communication UI Feature Component List.';
+  return undefined;
+  // throw 'Can\'t find corresponding selector for this component. Please check the supported components from Azure Communication UI Feature Component List.';
```

Comment:
What's the future plan for this error?
Wanna remove it completely?

Reply:
We remove it here because it is not an exposed api anymore (and we need this undefined to detect which context we are under)
But if you look at top layer mergedHooks.ts, you will see the error is throwing there